mmv1/templates/terraform/examples/bigquery_job_query_reservation.tf.tmpl

```diff
@@ -0,0 +1,12 @@
+resource "google_bigquery_job" "{{$.PrimaryResourceId}}" {
+  provider = google-beta
+
+  job_id     = "{{index $.Vars "job_id"}}"
+
+  query {
+    query = "SELECT state FROM [lookerdata:cdc.project_tycho_reports]"
+
+  }
+
+  reservation = "projects/{{index $.TestEnvVars "project"}}/locations/US/reservations/{{index $.Vars "reservation_name"}}"
```

Comment:
We'll ideally make a reservation and use it in our test- but currently have 0 slots on most/all test reservations. Will get back to you on this one.